amazon/iontest/ion_test_driver.py

```diff
@@ -181,7 +184,9 @@ def execute(self, *args):
             self._executable = os.path.abspath(os.path.join(self._build_dir, self._build.execute))
         if not os.path.isfile(self._executable):
             raise ValueError('Executable for %s does not exist.' % self._name)
-        _, stderr = Popen((self._executable,) + args, stderr=PIPE, shell=COMMAND_SHELL).communicate()
+        if self._prefix is None:
+            raise ValueError('Prefix for %s does not exist.' % self._name)
```

Comment:
I recommend handling this validation in the IonBuild constructor.

Reply:
I'm going to remove this validation since the prefix can be None for NO_OP_BUILD which is used for ion-test.